### ISSUE TITLE
Tally silently drops a dropdown option whose placeholder runs long: every instance's proposal form is missing phd, and no local check can see it

### ISSUE THREAD

**Issue by BenoitJT-GIRARD** [BenoitJT-GIRARD/convener#22]
Eighth of the family in #15 through #21, and the first that damages **data** rather than CI: **`phd` is missing from the proposal form of every instance this repository builds**, and nothing anywhere reports it.

## What happens

`tools/scripts/create_tally_form.py` builds Career stage as a DROPDOWN and puts the disambiguating gloss on the first option's `placeholder`. Tally accepts the block, exits 0, and **silently drops that option**. What it then serves:

```
DROPDOWN options, in document order:
  postdoc, independent, group-leader, other, undisclosed     <- 5 of 6, phd gone
  F, M, NB, undisclosed                                      <- 4 of 4
```

and the first career-stage option comes back renumbered, stripped of the group-level settings that rode with it:

```json
{"index":0,"isFirst":true,"isLast":false,"text":"postdoc"}
```

against Gender's, which survived intact:

```json
{"index":0,"isFirst":true,"isLast":false,"isRequired":false,
 "placeholder":"F (female), M (male), NB (non-binary), or undisclosed if you'd rather not say",
 "text":"F"}
```

The two blocks are structurally identical. The only material difference is length: **Career stage's gloss is 181 characters, Gender's is 77.** `DropdownOptionPayload.placeholder` is declared in Tally's own OpenAPI spec as `{"type": "string"}` with **no `maxLength`**, so nothing in the published contract warns of a limit, and the server enforces one anyway by discarding the whole option rather than truncating the field.

## What it costs

A PhD student cannot declare their career stage. They pick `other` or leave it, and `to_lead` records `other`/`undisclosed` — a legitimate-looking value, at a steady rate, in the field the cockpit's own Diversity screen reads. For a series whose proposers are largely doctoral researchers, the career-stage distribution is wrong from the first submission onward.

It also took the Career stage hint text with it: that question shipped with no explanation of `independent` versus `group-leader` at all, which is the second thing the gloss existed to prevent.

Both halves are silent. This instance's first real submission recorded `career_stage: undisclosed` and nothing at any layer said why.

## Why no test could see it

`test_the_career_stage_options_are_exactly_the_imported_vocabulary_in_order` passes, and passed throughout:

```python
assert _option_texts(build_blocks(), "Career stage") == list(CAREER_STAGE_ORDER)
```

It pins what the script *builds*. Nothing in the suite reads what Tally *accepts*, and the script's own exit code is 0 either way. The defect is only visible by fetching the published form and reading it back — which is how it was found, and how the fix below was verified.

## Two places the gloss must not go, and the second is a trap

**Not in the option `text`.** The module already forbids this, and is right: option text is the vocabulary token `to_lead` membership-tests.

**Not in the question's own wording** — and this one is worth stating plainly, because the module's docstring currently says it is where the gloss belongs:

> Each option's `text` is the bare vocabulary token [...] and nothing else: **any friendly gloss belongs in the question's own wording**, never smuggled into the option text

Read literally, that is a much worse defect than the one it would fix. The question's wording **is the matching key**: `convener_ops.journey.proposal` reads a submission by `label` against exact alternatives (`LABEL_CAREER_STAGE = ("Career stage", "Career level")`), never a prefix. Folding a gloss into the title renames the question, so every answer it ever receives fails to match and is recorded `undisclosed` — **for every respondent, not for a PhD student alone.** This was implemented, and `test_every_career_stage_dropdown_option_resolves_through_the_real_pipeline` caught it; without that test it would have shipped as a fix.

## The fix

A `TEXT` block, in a group of its own, emitted immediately before the question:

```python
gloss = _block("TEXT", "TEXT", f"{self.label}:gloss", {"html": self.placeholder})
return [gloss, title, *self._option_blocks()]
```

and no `placeholder` on the first option (`isRequired` stays).

Why there and nowhere nearer: `TitleBlock` is the only block type whose `groupType` admits `QUESTION` — `LabelBlock` is confined to `["LABEL"]` and `TextBlock` to `["TEXT","PAGE_BREAK","DIVIDER"]`. A TEXT group captures nothing, so it appears in no webhook payload and carries no label, which is exactly what lets it hold a long gloss without touching resolution.

## Verified, by the path that found the defect

Rebuilt against the live API, then re-read what Tally serves:

```
phd, postdoc, independent, group-leader, other, undisclosed     <- 6 of 6
F, M, NB, undisclosed                                           <- 4 of 4
TEXT gloss blocks served: both, the 181-character one intact
```

and round-tripped through a real submission:

| lead | chosen | recorded |
|---|---|---|
| `spk-001` | before the fix | `career_stage: undisclosed` |
| `spk-002` | `other` | `career_stage: other` |
| `spk-003` | `phd` | **`career_stage: phd`** |

Suite green: 4651 passed, 117 skipped.

## Suggested follow-up, beyond the fix

Two things this defect says about the shape of the checks rather than the code:

- The docstring sentence quoted above should be corrected where it stands, since it names the one location that breaks resolution entirely.
- Nothing reads the published form back. Every check here stops at the payload the script builds, and the one failure mode that matters — Tally accepting a block and quietly discarding part of it — lives past that boundary.

---

Found while standing an instance up by following `docs/operating/standing-up-for-an-agent.md`, on the step `proposal_form_live` check: the first real submission came back with `career_stage: undisclosed` and reading the served form explained why.

**Comment by BenoitJT-GIRARD** [comment]
Fixed on main by #39. The gloss rides in a TEXT block of its own rather than on the first option's `placeholder`: Tally accepted Gender's 77-character gloss there and silently dropped the whole option carrying Career stage's 181-character one, taking `phd` out of the published form while the script exited 0 and every test passed. It does not go into the question's wording either — `proposal.py` matches a submission by exact label, so a gloss there would unmatch every answer the question ever receives.
